tests/jballerina-debugger-integration-test/src/main/java/org/ballerinalang/debugger/test/utils/DebugOutputListener.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.debugger.test.utils;
+
+import org.ballerinalang.debugger.test.utils.client.TestDAPClientConnector;
+import org.eclipse.lsp4j.debug.OutputEventArguments;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.TimerTask;
+import java.util.concurrent.ConcurrentLinkedQueue;
+
+/**
+ * Timer Task implementation to capture debugger output from server output events.
+ */
+public class DebugOutputListener extends TimerTask {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DebugOutputListener.class);
+    TestDAPClientConnector connector;
```

Comment:
Shall we make the 'connector' private as it has a public getter?

Reply:
Made private in c15d1af